CRM/Upgrade/Incremental/php/FiveSeventySix.php

```diff
@@ -35,6 +35,26 @@ public function upgrade_5_76_alpha1($rev): void {
     $this->addTask('Alter translation to make string non-required', 'alterColumn', 'civicrm_translation', 'string',
       "longtext NULL COMMENT 'Translated string'"
     );
+    $this->addTask('Install SiteToken entity', 'createEntityTable', '5.76.alpha1.SiteToken.entityType.php');
+    $this->addTask(ts('Create index %1', [1 => 'civicrm_site_token.UI_name_domain_id']), 'addIndex', 'civicrm_site_token', [['name', 'domain_id']], 'UI');
```

Comment:
This line shouldn't be necessary. The index is already part of the entityType.php file and will be installed with the table.

Reply:
Nope I checked a bunch of times! Was gonna raise it with you - any way will merge this now it has passed & we can look again

Reply:
@eileenmcnaughton I put a breakpoint at the end of `generateCreateTableWithConstraintSql()` and the sql it runs to create the table does include the index, unless I'm missing something. It outputs:
```sql
CREATE TABLE IF NOT EXISTS `civicrm_site_token` (
  `id` int unsigned NOT NULL AUTO_INCREMENT,
  `domain_id` int unsigned NOT NULL,
  `name` varchar(64) NOT NULL COMMENT 'Token string, e.g. {site.[name]}',
  `label` varchar(255) NOT NULL COMMENT 'User-visible label in token UI',
  `body_html` text COMMENT 'Value of the token in html format.',
  `body_text` text COMMENT 'Value of the token in text format.',
  `is_active` boolean NOT NULL DEFAULT TRUE COMMENT 'Is this token active?',
  `is_reserved` boolean NOT NULL DEFAULT FALSE COMMENT 'Is this token reserved?',
  `created_id` int unsigned COMMENT 'Contact responsible for creating this token',
  `modified_id` int unsigned COMMENT 'FK to contact table.',
  `modified_date` timestamp NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT 'When the token was created or modified or deleted.',
  PRIMARY KEY (`id`),
  UNIQUE INDEX `UI_name_domain_id`(`name`, `domain_id`),
  CONSTRAINT `FK_civicrm_site_token_domain_id` FOREIGN KEY (`domain_id`) REFERENCES `civicrm_domain`(`id`) ON DELETE CASCADE,
  CONSTRAINT `FK_civicrm_site_token_created_id` FOREIGN KEY (`created_id`) REFERENCES `civicrm_contact`(`id`) ON DELETE SET NULL,
  CONSTRAINT `FK_civicrm_site_token_modified_id` FOREIGN KEY (`modified_id`) REFERENCES `civicrm_contact`(`id`) ON DELETE SET NULL
)
ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci ROW_FORMAT=DYNAMIC;
```

Reply:
@colemanw yeah it does seem to show   UNIQUE INDEX `UI_name_domain_id`(`name`, `domain_id`),